services/proxy/README.md

```diff
@@ -16,15 +16,4 @@ You need to set the letsencrypt options [here](./config/.tls.env).
 
 ## Enable TLS
 
-The proxy sets a default certificate resolver, using letsencrypt. To use it, you should:
-
-1. change the [resolver settings](./config/.tls.env). If the docker volume `letsencrypt_proxy_data` already exists, you might need to remove it to apply changes from the [.tls.env file](./config/.tls.env)
-2. add dedicated labels to each service to expose, making sure that the URLs are reachable by letsencrypt. You should set: the service public URL and the certificate resolver annotation and set the entrypoint to `websecure` to use port 443 only. For example, for the [frontend service](../frontend/compose.base.yaml):
```

Comment:
We should keep the information that letsencrypt requires a publicly resolvable and reachable hostname and ip for that hostname.

Reply:
true! thanks for the suggestion!